runtimes/alibi-explain/mlserver_alibi_explain/runtime.py

```diff
@@ -87,8 +87,16 @@ async def predict(self, payload: InferenceRequest) -> InferenceResponse:
             model_name=self.name,
             model_version=self.version,
             outputs=[output_data],
+            parameters=Parameters(headers={
+                "x-seldon-alibi-type":"explanation",
```

Comment:
do we want also to be specific about which type of explanation we are returning?

Reply:
That should come from the method header we return no?

Reply:
oh yes I thought we are just parsing the module of the mlserver runtime (i.e. `black_box_runtime` etc.) and not the alibi module. In this case then yes.

maybe then we could have a simple test case to make sure that these headers are set as expected?